internals/daemon/daemon_test.go

```diff
@@ -1148,3 +1148,35 @@ services:
 	c.Assert(tasks, HasLen, 1)
 	c.Check(tasks[0].Kind(), Equals, "stop")
 }
+
+func (s *daemonSuite) TestSyscallRebootDelay(c *C) {
+	waitState := 0
```

Comment:
These are booleans, right? I think you'd normally use `true` or `false` for this.

Reply:
Cleaned up. Sorry about the state of the tests - I rushed out the PR for overnight feedback, but the tests needed more attention :(